inspirehep/base/templates/format/record/Inspire_Default_HTML_brief_macros.tpl

```diff
@@ -18,8 +18,7 @@
 #}
 
 {% macro render_doi() %}
-  {% if record.get('dois')| is_list() %}
-    {% set filtered_doi = record.get('dois.value')| remove_duplicates() %}
+    {% set filtered_doi = record.get('dois.value')|tranform_nested_list()|remove_duplicates() %}
```

Comment:
This would be better solved in the `dojson` rule for dois:

```python
@hep.over('dois', '^024[1032478_][10_]')
def dois(self, key, value):
    """Other Standard Identifier."""
    value = utils.force_list(value)
    out = []
    for val in value:
        if val and val.get("2", '').lower() == "doi":
            out.append({
                'value': val.get('a'),
                'source': val.get('9')
            })
    return out
```

So that we never put a list inside of `value`, instead separate it in two different fields.